Library/Homebrew/cmd/vendor-install.sh

```diff
@@ -32,6 +32,24 @@ then
   esac
 fi
 
+if [[ -n "$HOMEBREW_LINUX" ]]
+then
+  LDD_VERSION_OUTPUT=$(/usr/bin/ldd --version)
+  if [[ $LDD_VERSION_OUTPUT =~ \ [0-9]\.[0-9]+ ]]
+  then
+    LDD_VERSION=${BASH_REMATCH[0]}
+    LDD_VERSION_MAJOR=${LDD_VERSION%.*}
+    LDD_VERSION_MINOR=${LDD_VERSION#*.}
+    if (( LDD_VERSION_MAJOR < 2 || LDD_VERSION_MINOR < 13 ))
+    then
+      odie "Vendored tools require system Glibc 2.13 or later"
+    fi
+  else
+    odie "Failed to detect system Glibc version"
+  fi
+  unset LDD_VERSION_OUTPUT LDD_VERSION LDD_VERSION_MAJOR LDD_VERSION_MINOR
```

Comment:
Could/should these be local variables in a function instead?

Reply:
Sure. I can convert this to a function and call it in `homebrew-vendor-install ()`.

Reply:
done